lib/profile.js

```diff
@@ -16,8 +16,7 @@ const usageUtil = require('./utils/usage.js')
 const usage = usageUtil(
   'npm profile enable-2fa [auth-only|auth-and-writes]\n',
   'npm profile disable-2fa\n',
-  'npm profile get [<key>]\n',
-  'npm profile set <key> <value>'
```

Comment:
not sure why this was removed, we definitely have `npm profile set` implemented

Reply:
I looked at other calls to usage.js. This call is missing trailing `+` -- it's not supposed to be a variadic function.

I also noticed that access.js incorrectly had its first argument as `'npm access'` instead of `'access'`.

Reply:
good catch, we definitely had the param to usage wrong here!